code/game/dna/mutations/disabilities.dm

```diff
@@ -536,3 +536,17 @@
 /datum/mutation/disability/dizzy/deactivate(mob/living/M)
 	..()
 	M.SetDizzy(0)
+
+/datum/mutation/disability/uwu
+	name = "Nya"
+	desc = "Makes you tawk a wittwe stwangewy."
+	activation_messages = list("Umm, it feels wike something is weally off... >.<")
+	deactivation_messages = list("Nyoo~ you feel all cutesy and normal again, huh? *purrs softly* it’s nice to feel all snuggly and comfy, meow~ :3 let’s keep it that way, okay? >///< nyaa~")
```

Comment:
the text you have is good, but deactivation messages are usually "hey, I'm okay now, here's how I'm feeling."

```suggestion
	deactivation_messages = list("You feel like people will take you a bit more seriously.")
```